client/web/src/cody/components/CodyMarketingPage/CodyMarketingPage.tsx

```diff
@@ -0,0 +1,231 @@
+]
+
+export interface CodyMarketingPageProps {
+    context: Pick<SourcegraphContext, 'authProviders'>
+}
+
+export const CodyMarketingPage: React.FunctionComponent<CodyMarketingPageProps> = ({ context }) => {
+    const { theme } = useTheme()
+    const isDarkTheme = theme === Theme.Dark
+
+    return (
+        <Page>
+            <PageTitle title="Cody AI" />
+            <PageHeader
+                description={
+                    <>
+                        Cody answers code questions and writes code for you by reading your entire codebase and the code
+                        graph.
+                    </>
+                }
+                className={classNames('mb-3', styles.pageHeader)}
+            >
+                <PageHeader.Heading as="h2" styleAs="h1">
+                    <PageHeader.Breadcrumb icon={CodyColorIcon}>
+                        <div className={classNames('d-inline-flex align-items-center', styles.pageHeaderBreadcrumb)}>
+                            Cody AI
+                        </div>
+                    </PageHeader.Breadcrumb>
+                </PageHeader.Heading>
+            </PageHeader>
+
+            {/* Page content */}
+            <div className={styles.headerSection}>
+                <div>
+                    <H1>Sign up to get Cody, our AI assistant, free</H1>
+                    <div className="ml-3">
+                        <Text className={styles.codyConversation}>Auto-generate unit tests</Text>
+                        <Text className={styles.codyConversation}>Explain code</Text>
+                        <Text className={styles.codyConversation}>Find code smells</Text>
+                        <Text className={styles.codyConversation}>Generate code</Text>
+                    </div>
+                    <CodyHelpIcon className={styles.codyHelpIcon} />
+                </div>
+
+                <MarketingBlock
+                    contentClassName={styles.codySignUpPanel}
+                    wrapperClassName={styles.codySignUpPanelWrapper}
+                >
+                    <H2>Sign up to get free access</H2>
+                    <Text className="mt-3">
+                        Cody combines an LLM with the context of Sourcegraph’s code graph on public code or your code at
+                        work. Sign up with:
+                    </Text>
+                    <div className={styles.buttonWrapper}>
+                        <ExternalsAuth
+                            context={context}
+                            githubLabel="Github"
+                            gitlabLabel="Gitlab"
+                            withCenteredText={true}
+                            onClick={onClickCTAButton}
+                            ctaClassName={styles.authButton}
+                            iconClassName={styles.buttonIcon}
+                        />
+                    </div>
+                    <Link to="https://sourcegraph.com/sign-in?returnTo=/search">Or, continue with email</Link>
```

Comment:
is the returnTo to search intentional here? cc @rrhyne

Reply:
No, good catch! I think this should be:

```suggestion
                    <Link to="https://sourcegraph.com/sign-up?showEmail=true">Or, continue with email</Link>
```